invoices_app/tests/test_detail_invoice.py

```diff
@@ -61,3 +62,19 @@ def test_get_detail_invoice_with_no_corresponding_supplier(self):
                 self.invoice_from_other_user.id
             ),
         )
+
+    def test_get_detail_invoice_with_new_comment(self):
+        self.client.force_login(self.ap_user)
+        self.invoice.new_comment_from_supplier = True
+        self.invoice.save()
+
+        self.client.get(
+            reverse(
+                'invoices-detail',
+                kwargs={
+                    'taxpayer_id': self.invoice.taxpayer.id,
+                    'pk': self.invoice.id,
+                }
+            )
+        )
+        self.assertFalse(Invoice.objects.filter(pk=self.invoice.id).last().new_comment_from_supplier)
```

Comment:
if you are retrieving one value from db you can use get instead of filter and removing last.
`Invoice.objects.get(pk=self.invoice.id).new_comment_from_supplier`

Reply:
Okas